resource-manager/src/main/java/org/ihtsdo/otf/resourcemanager/ResourceManager.java

```diff
@@ -302,6 +359,31 @@ public void moveResource(final String fromResourcePath,
 		}
 	}
 
+	public void copyResource(final String fromResourcePath,
+							 final String toResourcePath) throws IOException {
+		try {
+			if (resourceConfiguration.isUseCloud()) {
+				s3MoveResource(fromResourcePath, toResourcePath, false);
```

Comment:
@pgwilliams This "copy" function does not "delete" because of the negative boolean.